not loading, missing something.. :/

isao opened the pull request:
the ac/assets.common test won't load. arrow says:

```
[2012-07-22 17:25:30.304] [ERROR] SeleniumDriver - Failed to collect the test report
[2012-07-22 17:25:30.304] [ERROR] Arrow - Failed to collect the test report, Controller: /opt/brew/lib/node_modules/arrow/lib/controller/default
```

don't merge.. but pointers appreciated

Commit:
Merge branches 'develop' and 'master' - rel 0.3.8

drewfish committed:
removed deprecation of autoload/ because yui_modules/ isn't ready yet

Commit:
merge release 0.3.25 (based on 0.3.24-34-g998bca1)

jlecomte committed:
Allow user-provided middlewares with a name starting with 'mojito-' to gain access to the resource store, YUI instance, logger, context, etc.

Commit:
merge release 0.3.26 (based on 0.3.25-2-g380d8bc)

Commit:
Merge pull request #187 from jlecomte/master

Allow user-provided middleware with a name starting with 'mojito-' to gain access to the resource store, YUI instance, logger, context, etc.

Commit:
merge release 0.3.27 (based on 0.3.26-71-gc53e3f0)

idearat committed:
merge release 0.3.28 (based on 0.3.27-41-g4ee1a78)

drewfish committed:
merge release 0.3.29 (based on 0.3.28-9-gaafd34b)

Commit:
Added unit tests and fixes for mojit-proxy.client.js

Commit:
Merge branch 'master' of github.com:yahoo/mojito into arrow

Commit:
Fixed up descriptor; Fixed tests not working under nodejs driver

Commit:
Core arrow test infrastructure.

Commit:
De-lint test files.

add0n committed:
Merge pull request #290 from mojit0/arrow

Arrow core infrastructure.

Commit:
Added tests for output-handler.client.js and fixed potential breakages

Commit:
Merge pull request #296 from mridgway/output-handler.client

Added tests for output-handler.client.js and fixed potential breakages

Commit:
added missing mock verification

Commit:
added mojito-tunnel-client unit tests

Commit:
Merge pull request #297 from mridgway/arrow

Tunnel Client Unit Tests

mojit0 committed:
Fix unit test baseline glitches.

mojit0 commented:
That's a lot of commits. What's the specific set of deltas that's triggering this issue?

isao commented:
oops, weird. side note github improved the pull request handling to land pr's on any fork/branch.. cool.

anyway resynced. been trying a bunch of things, but the simplest case is the dataprovider "assets.common"..

Commit:
Ported analytics ac addon tests to arrow

Commit:
Merge branch 'arrow' of github.com:yahoo/mojito into arrow

Commit:
Merge pull request #312 from mridgway/arrow

Ported analytics ac addon tests to arrow

Commit:
Moved descriptors to leaf directories

mojit0 committed:
Merge pull request #313 from mridgway/arrow

Moved test descriptors to leaf directories

mojit0 committed:
Shortened dependency paths via config data.

Updated config property content to avoid repeated use of ../.. in test
descriptions.

mojit0 committed:
Shortened dependency paths via config data.

Updated config property content to avoid repeated use of ../.. in test
descriptions.

mojit0 committed:
Fix unit test baseline glitches.

Updated paths in README, adjusted YUI references in test files.

mojit0 committed:
Fix unit test baseline glitches.

Updated paths in README, adjusted YUI references in test files.

mojit0 committed:
Removed conflicting file.

mojit0 committed:
Merge branch 'arrow' of github.com:mojit0/mojito into arrow

mojit0 committed:
Rebase and discard changes.

mojit0 committed:
Repair README and mojito-test.html path references.

README pointed to obsolete file name.
mojito-test.html referenced uncommitted files in a comment.

Commit:
Merge pull request #314 from mojit0/arrow

Repair base test file content.
